Assets/MRTK/Core/Inspectors/Profiles/MixedRealitySceneSystemProfileInspector.cs

```diff
@@ -311,5 +319,26 @@ private void DrawContentSceneElement(Rect rect, int index, bool isActive, bool i
         {
             SceneInfoDrawer.DrawProperty(rect, contentScenes.GetArrayElementAtIndex(index), GUIContent.none, isActive, isFocused);
         }
+
+        private bool FindDefaultResources()
+        {
+            string[] defaultResourceGuids =
+            {
+                "ae7bb08d297fb69408695d8de0962524", // DefaultManagerScene
+                "7e54e36c44f826c438c95da79f8de638"  // DefaultLightingScene
+            };
+
+            bool findSucceeded = true;
+            foreach (string s in defaultResourceGuids)
+            {
+                if (AssetDatabase.LoadAssetAtPath<SceneAsset>(AssetDatabase.GUIDToAssetPath(s)) == null)
```

Comment:
I think my only concern would be if this is expensive to do twice every GUI refresh in `OnInspectorGUI` (can we cache it in `OnEnable`? Which should get called if the asset database is refreshed and these scenes might be added / removed), but if it doesn't seem to slow anything down that should be fine.

Reply:
i have not noticed any slowdown. doing this here also ensures that the inspector does not need to be closed and reopened for the check to be performed.

Reply:
`OnEnable` _should_ fire for open windows/inspectors when the database is refreshed. That's been my experience with windows, but maybe inspectors behave differently?

Reply:
in that case, i can try changing that :)

Reply:
+1 to @keveleigh's comment

Reply:
i set breakpoints and am NOT getting any repeat calls into OnEnable after deleting the files (from the command line) or re-entering the profile (switch to a different inspector and/or a different object)

Reply:
Would it be possible to cache the LoadAssetAtPath result and see if it goes `null` when the corresponding scene is deleted?

Reply:
That way we're at least more performant in the "both scenes exist" case, but the same in the "one or neither scenes exist" case.

Reply:
looks like that's a good fix... thanks!